android/build.gradle

```diff
@@ -0,0 +1,61 @@
+apply plugin: 'com.android.application'
+
+def hostBuildDir = new File(rootProject.buildDir, "host")
```

Comment:
I'm honestly very uneducated about Android builds and have no idea what's going on here, but you seem to know what you're doing

Reply:
yeah, this is all just super-standard android app stuff.